var/spack/repos/builtin/packages/python/package.py

```diff
@@ -162,7 +162,7 @@ class Python(AutotoolsPackage):
         # system-provided version is sufficient to build Python's UUID support,
         # the easy solution is to only depend on Spack's libuuid when *not* on
         # a Mac.
-        depends_on('libuuid', when='+uuid')
+        depends_on('uuid', when='+uuid')
```

Comment:
Can you confirm that `python+uuid` builds correctly with a few different `uuid` providers on Linux? I'm planning on making `python+uuid` the default in #18252.

Reply:
I did a test build but of course forgot to actually specify `+uuid`. :man_facepalming: I just pushed a new version that should fix this.